index_data.py

```diff
@@ -12,7 +12,10 @@
 currentpath = os.path.dirname(os.path.abspath(__file__))
 indexpath = os.path.join(currentpath, 'index')
 
-
+if os.sep == '\':
```

Comment:
This breaks on windows with EOF while scanning literal.
``` if os.sep == '\': ```
This can not work with windows. 
I don't know about Linux / mac but I am getting an error.
Please use platform detection method.

![Screenshot (1)](https://user-images.githubusercontent.com/23142350/54512171-56582b00-4979-11e9-9c14-4c799e04d4a3.png)